### PR TITLE
AirportProject

### PR DESCRIPTION
**Pull request by Ibraga96**
Didn't do DataAnnotations , because want to leave IDs in project and change them to "numbers " column.

### CODE REVIEW

**Review by AnatoliyDubovtsev — APPROVED** [review]
This is looking good

**Comment by Ibraga96** [comment]
> This is looking good

Thanks) I think there is no mistake with the branching now, yes?)

File: UtopiaCity/Controllers/Airport/FlightController.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+namespace UtopiaCity.Controllers.Airport
+{
+    public class FlightController:Controller
+    {
+        private FlightService _flightService;
+        public FlightController(FlightService flightService)
+        {
+            _flightService = flightService;
+        }
+        public IActionResult Index()
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Airport/FlightController.cs:21]: Empty lines between methods will make code look better. Same thing throughout whole file/files

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Controllers/Airport/FlightController.cs (hunk 1 above)
Comment (Ibraga96) [reply UtopiaCity/Controllers/Airport/FlightController.cs:21]: OK will fix it

**Comment by Ibraga96** [comment]
I modified previous commit , and added new service pls can you check again?

**Pull request merged**